StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/AccountSessionClaimedMessageHandler.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AccountSessionClaimedMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+import Foundation
+
+// Emitted when the account session has been created.
+class AccountSessionClaimedMessageHandler: ScriptMessageHandler<AccountSessionClaimedMessageHandler.Message> {
+    struct Message: Codable, Equatable {
+        let merchantId: String
```

Comment:
nit: should we copy the comments forward from the doc?
```suggestion
        /// The connected account ID
        let merchantId: String
```